### PR TITLE
[9.0][ADD][website_sale_custom_info] New module to show technical datasheets.

### PR DESCRIPTION
**Pull request by yajo**
Combines the power of `product_custom_info` and `website_sale`.

The resulting page is similar to:

captura de pantalla de 2016-08-11 18-19-11

WIP until merged:
- [x] https://github.com/OCA/product-attribute/pull/177
- [x] https://github.com/OCA/server-tools/pull/492

@Tecnativa

### CODE REVIEW

File: website_sale_custom_info/controllers/main.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+                    ("value", "=", value),
+                ])
+                domains.setdefault(prop, list())
+                domains[prop].append(("custom_info_ids", "in", records.ids))
+            # Join several critera for same property by OR
+            for domain in domains.values():
+                result += ["|"] * (len(domain) - 1) + domain
+        return result
+
+    @route()
```
Comment (lasley) [inline website_sale_custom_info/controllers/main.py:38]: Is this accidental, or some sort of wizardry I am not privy to?

Comment (yajo) [inline website_sale_custom_info/controllers/main.py:38]: Option 2. See https://www.odoo.com/documentation/9.0/reference/http.html#controllers

Comment (lasley) [inline website_sale_custom_info/controllers/main.py:38]: Oh that's awesome, thanks for the elaboration @Yajo

**Comment by yajo** [comment]
After last commit, you can now use custom info as search filters!

Quick filters:

captura de pantalla de 2016-08-16 17-21-32

Advanced filters:

captura de pantalla de 2016-08-16 17-21-45

And you can choose several values per property! :tada:

**Comment by pedrobaeza** [comment]
Closing this due to no continuation in the project. If somebody has interest on it, he/she can open another PR taking this work.

**Comment by Rad0van** [comment]
This is very interesting to me - I need something exactly like this :) @pedrobaeza and @Yajo - is there a specific reason you stopped developing this? Is there any alternative? I am asking just to get the facts before I try to pick this up and continue with it...

**Comment by pedrobaeza** [comment]
The customer finally didn't launch this, but you can take it as is and migrate it to new versions.

**Comment by Rad0van** [comment]
> The customer finally didn't launch this, but you can take it as is and migrate it to new versions.

Thanx for quick reply. Will try to do that. But before I have to solve one thing  with base_custom_info. I'll ask in that module - maybe you could suggest a way.

**Pull request closed without merging**